src/smooch-node.js

```diff
@@ -64,6 +65,7 @@ export class Smooch extends SmoochBase {
             this.appUsers = new AppUsersApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
             this.conversations = new ConversationsApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
             this.stripe = new StripeApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
+            this.attachments = new AttachmentsApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
```

Comment:
That seems to work just fine... Must have not understood it correctly when I originally tried that.